### ISSUE TITLE
NTR: Dihydroxyglutarate high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8868]
New term request Dihydroxyglutarate high in urine (elthat_151002143027): DHG

**Comment by pnrobinson** [comment]
@MickeySegal
what is the context?
Is this 
D-2-hydroxyglutaric acid? https://omim.org/entry/600721
--

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK316514/
"dihydroxyglutarate (DHG)"
There is also a mention of
"2,4-dihydroxy-glutamate (DHG)"
but based on https://link.springer.com/article/10.1007/s00240-021-01290-2 I think the first is correct.  
I wrote to Margaret Adam at GeneReviews pointing out the inconsistency.

**Comment by pnrobinson** [comment]
@MickeySegal 
Then it is probably this.
OMIM does not seem to list the finding.
[2-hydroxyglutarate aciduria HP:0032278](https://hpo.jax.org/browse/term/HP:0032278)

**Comment by pnrobinson** [comment]
Let me check

**Comment by pnrobinson** [comment]
Here is a clinical test
https://testdirectory.questdiagnostics.com/test/test-detail/12730/24-dihydroxyglutarate-urine-with-creatinine?p=r&cc=MASTER

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001081
- label: Elevated urinary 2,4-dihydroxyglutarate level
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: PMID:34821949
- def: The amount of 2,4-dihydroxyglutarate in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8868
- synonyms:
